message-center.js

```diff
@@ -22,10 +39,11 @@ MessageCenterModule.
           /** @var Do not delete this message automatically. */
           permanent: 'permanent'
         },
-        add: function (type, message, options) {
+        add: function(type, message, options) {
```

Comment:
Let's not remove the spaces from anonymous function declarations, those are on purpose.